### PR TITLE
[GameINI] Netplay Smash AR codes

### PR DESCRIPTION
**Pull request by Helios747**
Fixes blocker https://bugs.dolphin-emu.org/issues/9569 - Please tag as appropriate

These are the AR codes the smash community has put together and has requested be included in 5.0

Dansalvato in #Dolphin-dev is the point of contact on this. @JMC47 can probably vouch for this anyways.

---
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3876)

### CODE REVIEW

**Comment by JMC47** [comment]
Smash ladders testing should be more than enough.  These codes are mostly
for the smash community after all.  Lgtm.

**Comment by Helios747** [comment]
Don't merge this yet. I noticed there were both AR and gecko in the last version and the version dansalvato gave didn't label either. Need to find out from them which are which

**Comment by Helios747** [comment]
This is now mergable. Dansalvato clarified in IRC what they wanted.

File: Data/Sys/GameSettings/GALE01r2.ini [hunk 1]
```diff
@@ -396,66 +432,107 @@ $Zelda is Permanently Sheik at CSS [Achilles]
...
-8210001C 82100008
-9A900024 1E12001C
-3A100008 7E70A9AE
-3A00001E 9A160000
-2C110001 41820010
-3A520001 2C12001D
-41A0FF4C 39EF0001
-2C0F0005 41A0FEE8
-C022C9E8 00000000
+**Banned stages canÂ´t be selected anymore, however they can still be selected when someone chooses "Random".
```
Comment (JosJuice) [inline Data/Sys/GameSettings/GALE01r2.ini]: It doesn't matter much, but could you use a regular ' here instead of ´?

**Comment by JosJuice** [comment]
Maybe it would be nice to have the input delay reduction available as a separate code, not just as part of the netplay code, but that's not a reason to hold this PR back. It can always be added later.

File: Data/Sys/GameSettings/GALE01r2.ini (hunk 1 above)
Comment (Helios747) [inline Data/Sys/GameSettings/GALE01r2.ini]: Done.

**Comment by IDGeek121** [comment]
Hi. Just built this for myself, and I noticed that the Gecko code header (not sure if that's what it's called) was removed. I think the Netplay Community Settings code is a Gecko code, and not an AR code. The code didn't work when i toggled it in the AR window. However, when I added the [Gecko] heading back to line 144 and toggled the Netplay Community Settings code in the Gecko window, it worked perfectly.

**Comment by Helios747** [comment]
I won't be able to fix this for at least 7 hours. If somebody wants to get a PR to fix this before hand feel free and I apologize, otherwise I'll fix this tonight.

**Comment by Helios747** [comment]
It's already been made.

**Comment by RisingFog** [comment]
@CoolOppo already done with #3877

**Comment by CoolOppo** [comment]
Yeah I saw :)

**Pull request merged**